spec/symbols_spec.rb

```diff
@@ -43,4 +51,31 @@ def expect_transformation(klass, *input, arg_signature)
 
     expect_transformation(SampleJob, *input, arg_signature)
   end
+
+  it "works with subclasses that don't include the module directly" do
+    input = [1, "x" => { "y" => 2, z: { "foo bar" => 0 } }]
+    arg_signature = [[1], { x: { y: 2, z: { :"foo bar" => 0 } } }]
+
+    expect_transformation(SubclassWithoutIncludeJob, *input, arg_signature)
+  end
+
+  if RUBY_VERSION >= "2.1.0"
```

Comment:
It's trying to explain that `perform_async` won't actually `raise`. The `raise` from a missing required keyword argument will happen at the place where `FooJob.new.perform` happens in the Sidekiq server, not on the client where `perform_async` is called.